### ISSUE TITLE
Different appearances for the SmartYouTubeTV app on two SHIELD TVs

### ISSUE THREAD

**Issue by H2SO4JB** [yuliskov/SmartYouTubeTV#367]
Hi,

I want to start by thanking you for a great app that I use on my two SHIELD TVs.

As I said, I have two SHIELD TV's that I just installed SmartYouTubeTV on. I have the latest version of everything installed on both. However, I get different appearances for the SmartYouTubeTV app on my two SHIELD TVs. I show what I mean with two screenshots below.

Can anyone here explain why? Did I accidentally click on any hidden setting somewhere? What I can see under SmartYouTubeTV settings is that my settings are identical on both my SHIELD TVs.

This is very annoying. Please, can anyone explain?

![TV1](https://user-images.githubusercontent.com/13471707/67691937-b0a17b80-f99f-11e9-8cb4-bded2e519d9c.jpg)
![TV2](https://user-images.githubusercontent.com/13471707/67691947-b39c6c00-f99f-11e9-9292-83edb366a91b.jpg)

**Comment by H2SO4JB** [comment]
I can add that I now found this information:
https://www.androidpolice.com/2019/10/03/youtube-for-android-tv-gets-a-light-makeover-with-expanded-menu-options/

But this is for the official YouTube app. Is SmartYouTubeTV also affected by this change? And if so, how do I force an update to the new look on both my SHIELD TVs? This is so confusing...

**Comment by yuliskov** [comment]
Hi. Try this build. And note, that you need to install the app from scratch (without old data).
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.199/SYTV_Orig_v6.17.199_r.apk

**Comment by H2SO4JB** [comment]
Hi,
Thanks, I have now reinstalled the app on both my SHIELD TVs. Now I have the old UI on both. For me, it doesn't matter what UI it is, the important thing is that it is the same on both.  :-)

**Comment by yuliskov** [comment]
Thanks for testing.

**Comment by fgl27** [comment]
I made some minor improves to the new UI

In case you are interested @yuliskov as yo add a switch to choose what to use the new is somewhat unfinished...

https://github.com/fgl27/SmartYouTubeTV/blob/new-exo-player/smartyoutubetv/src/main/assets/addons/on_load/00_ui_tweaks/00_ui_tweaks.css

Before

![0001](https://user-images.githubusercontent.com/13755892/67787388-42b48d00-fa4f-11e9-82c0-b6a40c29273f.png)

After

![0002](https://user-images.githubusercontent.com/13755892/67787395-46e0aa80-fa4f-11e9-9363-b0d599e66604.png)

Before

![0003](https://user-images.githubusercontent.com/13755892/67787407-4ba55e80-fa4f-11e9-9e06-8ca46120f7f8.png)

After

![0004](https://user-images.githubusercontent.com/13755892/67787418-519b3f80-fa4f-11e9-9a9b-e0a574b8539c.png)

Basically makes the subscriber screen hold two complete rows of content without cutting thumbnail info and allows the thumbnails title to show full video title (or almost full if some title are tooooo big I guess didn't see one too big yet that didn't fit)

Don't know if that affect all devices but all my are affected.
I wanna to fix the main window (home and etc) to hold two full row of content but didn't manage to fix it.

I wish google didn't use interns to make they UI :laughing: after all is so unfinished.

**Comment by fgl27** [comment]
Btw seems that there is more then one new UI version as on my devices I have a diff side menu order when using official youtube app...
Maybe is just a order difference but maybe also has others changes.

see

![0001](https://user-images.githubusercontent.com/13755892/67788228-bacf8280-fa50-11e9-963c-60d6fde4b53a.png)

:+1:

**Comment by yuliskov** [comment]
@fgl27 thx friend. I'll check it.

**Comment by yuliskov** [comment]
@fgl27 one more question, friend. Have you noticed large ad banner on the new ui?

**Comment by fgl27** [comment]
No I guess my region doesn't have it yet. Never see it not even on official version.

**Comment by anubis-genix** [comment]
My UI switched to the new one yesterday without applying an update. I was quite surprised ... I'd love to have an option to switch between the old and new UI.

**Comment by yuliskov** [comment]
Seems, that Google disabled old ui completely.
